kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/index.vue

```diff
@@ -324,6 +324,12 @@
         if (!Number.isInteger(this.numQuestions)) {
           return this.$tr('numQuestionsBetween');
         }
+        if (this.availableQuestions == 0 || this.availableQuestions == null) {
+          return this.$tr('numQuestionsExceedNoExercises', {
```

Comment:
You don't need to add this new `numQuestionsExceedNoExercises` message. You can reuse the existing `numQuestionsExceed` message like so:

```
this.$tr('numQuestionsExceed', {
            inputNumQuestions: this.numQuestions,
            maxQuestionsFromSelection: 0,
});
```

Reply:
Interestingly when I tried that I was still seeing the issue. Also the the end of the string didn't match with what the issue asked for (they didn't want to say lower the number to 0). If we still want to change it though I can try again, just let me know!